src/server/api/user/index.ts

```diff
@@ -215,7 +214,6 @@ router.patch(
           }
         : undefined
 
-      // TODO: Remove force cast once UserRepository has been made
```

Comment:
@JasonChong96 wanted to confirm with you whether the comments deleted in this commit is okay

Reply:
They can be removed, I forgot to remove them in the previous pr